daal4py/sklearn/test/test_run_to_run_stability_tests.py

```diff
@@ -41,295 +40,269 @@
-
-    @unittest.skip("unstable")
-    def test_lasso_regression(self):
-        run_reg_test(run_lasso_regression)
-
-    @unittest.skip("unstable")
-    def test_pca_full(self):
-        run_clf_test(run_pca_full)
-
-    @unittest.skip("unstable")
-    def test_pca_daal4py_corr(self):
-        run_clf_test(run_pca_daal4py_corr)
-
-    #---------------------Expected to be fixed in next release---------------------
-
-    @unittest.skip("unstable")
-    def test_log_regression_newton(self):
-        run_clf_test(run_log_regression_newton)
-
-    @unittest.skip("unstable")
-    def test_log_regression_lbfgs(self):
-        run_clf_test(run_log_regression_lbfgs)
-
-    @unittest.skip("unstable")
-    def test_log_regression_cv_newton(self):
-        run_clf_test(run_log_regression_cv_newton)
-
-    @unittest.skip("unstable")
-    def test_log_regression_cv_lbfgs(self):
-        run_clf_test(run_log_regression_cv_lbfgs)
-
-    @unittest.skip("unstable")
-    def test_linear_regression(self):
-        run_reg_test(run_linear_regression)
-
-    @unittest.skip("unstable")
-    def test_ridge_regression(self):
-        run_reg_test(run_ridge_regression)
-
-    @unittest.skip("unstable")
-    def test_rf_class(self):
-        run_clf_test(run_rf_class)
-
-    @unittest.skip("unstable")
-    def test_rf_reg(self):
-        run_reg_test(run_rf_regression)
-
 
-if __name__ == '__main__':
-    unittest.main()
+MODELS_INFO = [
+    #----------------------Passed----------------------
+    {
+        'model_type'    : 'clf', 
+        'model_name'    : 'KNN_brute_uniform', 
+        'model'         : KNeighborsClassifier(n_neighbors=10, algorithm='brute', weights="uniform"),
+        'attributes'    : [],
+        'methods'       : ['predict', 'predict_proba', 'kneighbors'],
+    },
+    {
+        'model_type'    : 'clf', 
+        'model_name'    : 'KNN_brute_distance', 
+        'model'         : KNeighborsClassifier(n_neighbors=10, algorithm='brute', weights="distance"),
+        'attributes'    : [],
+        'methods'       : ['predict', 'predict_proba', 'kneighbors'],
+    },
+    {
+        'model_type'    : 'clf', 
+        'model_name'    : 'KNN_kd_tree_uniform', 
+        'model'         : KNeighborsClassifier(n_neighbors=10, algorithm='kd_tree', weights="uniform"),
+        'attributes'    : [],
+        'methods'       : ['predict', 'predict_proba', 'kneighbors'],
+    },
+    {
+        'model_type'    : 'clf', 
+        'model_name'    : 'KNN_kd_tree_distance', 
+        'model'         : KNeighborsClassifier(n_neighbors=10, algorithm='kd_tree', weights="distance"),
+        'attributes'    : [],
+        'methods'       : ['predict', 'predict_proba', 'kneighbors'],
+    },
+    {
+        'model_type'    : 'clf', 
+        'model_name'    : 'DBSCAN', 
+        'model'         : DBSCAN(algorithm="brute", n_jobs=-1),
+        'attributes'    : ['core_sample_indices_', 'components_', 'labels_'],
+        'methods'       : ['fit_predict'],

```

Comment:
We could check all attributes automatically. See [here](https://stackoverflow.com/questions/1911281/how-do-i-get-list-of-methods-in-a-python-class)